tests/kani/Intrinsics/AlignOfVal/align_of_basic.rs

```diff
@@ -0,0 +1,44 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+
+// Check that we get the expected results for the `min_align_of_val` intrinsic
+// with common data types
```

Comment:
Is this architecture specific, of do these min aligns come from the rust spec?

Reply:
My understanding is that `min_align_of_val` depends on `min_global_align`, which is an architecture option.
Please note that in #1054 we added checks to ensure that `min_global_align` is unspecified or equal to 1, otherwise Kani will stop there.

Reply:
Can you add a comment that the tests here assume x86, otherwise good to go.